doc/manual/packages.rst

```diff
@@ -570,11 +570,21 @@ going to have to do a *force push*:
   indicates that this should replace the ``fixbar`` branch found at
   ``myfork``.
 
-
 Creating a new Package
 ----------------------
 
-Guidelines for Naming a Package
+REQUIRE speaks for itself
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should have a ``REQUIRE`` file in your package repo, with a bare minimum directive of what Julia version to include.
+When Julia version ``0.x`` is released, use ``julia 0.x`` in your ``REQUIRE``. You may consider using ``julia 0.y-`` at this
```

Comment:
I was letting the `x` and `y` confuse me. Okay, I get it. By saying `0.y-` you are _not_ supporting `0.x`.